pulp_rpm/app/tasks/synchronizing.py

```diff
@@ -1028,8 +1028,8 @@ async def on_package(pkg):
                 Args:
                     pkg (createrepo_c.Package): A completed createrepo_c package.
                 """
-                if self.mirror and pkg.location_base:
-                    raise ValueError(XML_BASE_AND_MIRROR_INCOMPATIBLE_ERR_MSG)
+                if self.mirror and pkg.location_base or ".." in pkg.location_href:
```

Comment:
Should pulpcore do an additional check for this somewhere the creation chain for PublishedArtifact?